### PR TITLE
tmux: enable sixel support

### PR DESCRIPTION
**Pull request by laggardkernel**
#### Description

tmux added SIXEL support since 3.4, enable this feature during build.

https://github.com/tmux/tmux/blob/2df15ad08c9e4cf286cdb38cb645fc9066c3098d/CHANGES#L102

test method, build and open shell in tmux, install ImageMagick

```
convert /path/to/pic sixel:-
```

You should get a preview of the picture within tmux, if your terminal also support SIXEL images. (terminals including iTerm2, wezterm, etc)

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 12.7 21G816 arm64
Command Line Tools 14.2.0.0.1.1668646533

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`? (without -t trace mode, which is broken)
- [x] tested basic functionality of all binary files?
- [x] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by casr** [comment]
What would be the result with terminals that do not support Sixel such as Terminal.app or Alacritty?

Should it be applied as a variant instead?

**Comment by tessus** [comment]
I agree, this is the same as other escape sequences being sent by tmux natively. If the terminal does not understand them, it depends on the terminal to handle this situation.

TL;DR
Terminals are notorious for ignoring standards and/or extending the functionality without proper specs. In a perfect world, apps could query the terminal whether it supports certain features and in many cases this even works. However, it is not something an app can rely on completely. Thus there are many workarounds in a lot of apps to support multiple terminals seemlessly.

**Comment by casr** [comment]
I'll argue for a variant because:

1. `tmux` treats this as a feature that must be enabled
2. it's suffered resistance in the past with some projects adopting it due to things like performance concerns or 'why this standard over another?'

For those reasons, I think having the ability to toggle this as a variant would be welcomed by some.

I do not see a problem making this a default variant though.

**Review by casr — APPROVED** [review]

**Comment by reneeotten** [comment]
@tessus so is this good to merge?

**Comment by tessus** [comment]
@reneeotten I don't have merge access. someone from the macports team has to merge this

**Comment by reneeotten** [comment]
> @reneeotten I don't have merge access. someone from the macports team has to merge this

I understand that and I was going through PRs to merge things. But I just wanted to make sure you approve of these changes as that wasn't completely clear to me.

**Comment by tessus** [comment]
Oh, I see. Yes, LGTM and adding a variant makes sense. Thx.
